local-volume/bootstrapper/bootstrapper.go

```diff
@@ -344,13 +344,11 @@ func main() {
 	if config, err = ensureVolumeConfig(client, namespace); err != nil {
 		glog.Fatalf("Unable to ensure volume config: %v", err)
 	}
-
```

Comment:
Can you revert the whitespace changes to this file?

Reply:
done